content/en/flux/cheatsheets/oci-artifacts.md

```diff
@@ -396,6 +396,87 @@ Note that keyless verification is an **experimental feature**, using
 custom root CAs or self-hosted Rekor instances are not currently supported.
 {{% /alert %}}
 
+### Notary Workflow example
+
+Add a certificate to the trusted certificates store:
+
+```shell
+notation cert add --type ca --store <store> <cert>
+```
+
+It is also possible to generate a test certificate:
+
+```shell
+# Generate a certificate and RSA key pair
+notation cert generate-test valid-example
+```
+
+Push and sign the artifact using the certificate's private key:
+
+```shell
+flux push artifact oci://ghcr.io/stefanprodan/manifests/podinfo:$(git tag --points-at HEAD) \
+	--path="./kustomize" \
+	--source="$(git config --get remote.origin.url)" \
+	--revision="$(git tag --points-at HEAD)/$(git rev-parse HEAD)"
+
+notation sign ghcr.io/stefanprodan/manifests/podinfo:$(git tag --points-at HEAD) -k <key>
+```
+
+Create a `trustpolicy.json` file:
+
+```json
+{
+    "version": "1.0",
+    "trustPolicies": [
+        {
+            "name": "<policy-name>",
+            "registryScopes": [ 
+                "ghcr.io/stefanprodan/manifests/podinfo"
+             ],
+            "signatureVerification": {
+                "level" : "strict" 
+            },
+            "trustStores": [ "ca:<store-name>" ],
+            "trustedIdentities": [
+                "x509.subject: C=RO, ST=BU, L=Bucharest, O=Notary, CN=stefanprodan.com"
```

Comment:
Sorry, I had another look and realised you are resigning the artifact using your own key. Seeing stefanprodan's `podinfo` threw me off. Generating certs using `notation cert generate-test valid-example` will result a different `x509.subject` (`C=US, ST=WA, L=Seattle, O=Notary, CN=valid-example`). In my original PR for signing `podinfo` I added information around generating your own keys using `openssl` https://github.com/stefanprodan/podinfo/pull/341.

Perhaps it might be worth including a section detailing how to verify how to verify an externally signed artifact using the producer's certs?

Reply:
I agree, let's not use podinfo here at all but some generic `ghcr.io/org/app-manifests` with a generic trust store example. We can also link to at the end to podinfo `.notation` dir for a full example.